embulk-standards/src/main/java/org/embulk/standards/JsonParserPlugin.java

```diff
@@ -57,6 +59,10 @@ public String getString() {
         @Config("invalid_string_escapes")
         @ConfigDefault("\"PASSTHROUGH\"")
         InvalidEscapeStringPolicy getInvalidEscapeStringPolicy();
+
+        @Config("json_pointer_to_root")
```

Comment:
As discussed f2f, we may change the configuration name later before finalizing the release of this feature. And, we may release Embulk before the finalization.

What do you think:
* Add TODO comment to rename it, and
* Name this configuration so that it is clear this is still "experimental under development": for example `__experimental__json_pointer_to_root` ?

Reply:
That makes sense, I will do that.